package.json

```diff
@@ -38,7 +38,6 @@
     "regenerator-runtime": "^0.13.9",
     "register-service-worker": "^1.7.2",
     "sbc-common-components": "3.0.5",
-    "v-mask": "^2.3.0",
```

Comment:
We no longer need this.

However, we still need to import `vue2-filters`, below, because SbcFeeSummary, which is compiled in THIS project, needs it 😒